tests/e2e/solidity/suites.go

```diff
@@ -42,6 +41,10 @@ func stopSubnet() {
 
 var _ = utils.DescribePrecompile(func() {
 	ginkgo.It("tx allow list", func() {
+		if utils.GetMode() != "test" {
```

Comment:
Should the string `"test"` be a constant if we use it this way?

Reply:
Is there a better way to separate these tests than to need to add an if condition to the start of each test?

Can we perhaps define two different test suites and select one to execute based off of the environment variable as opposed to needing to handle this in each test function like this?